src/sql/workbench/contrib/queryplan2/browser/media/queryPlan2.css

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.actionbar-container {
+	height: 100%;
+	width: 30px;
+	position: absolute;
+	top: 0px;
+	right: 0px;
+}
+
+.qp2-container {}
+
+.query-plan2-container {
+	width: 100%;
+	height: 500px;
+	position: relative;
+	border-top: 1px solid;
+}
+
+.properties-container {
+	position: absolute;
+	top: 0px;
+	right: 32px;
+	height: 100%;
+	overflow-y: scroll !important;
+	background-color: #eeeeee;
+	width: 510px;
+}
+
+.prop-table-header {
+	font-weight: bold;
+}
+
+.properties-header {
+	font-weight: bold;
+	padding: 5px;
+}
+
+.toggle {
+	height: 9px;
+	width: 9px;
+	display: inline-block;
+}
+
+.toggle.expand {
+	background: url(../images/expand.gif) no-repeat center center;
+}
+
+.toggle.collapse {
+	background: url(../images/collapse.gif) no-repeat center center;
+}
+
+.mxTooltip {
```

Comment:
What's this class from?

Reply:
this is a style from the graph control and needs to have this name.